doc/ddu-source-git_branch.txt

```diff
@@ -107,6 +107,14 @@ delete
 deleteForce
 	Delete the branch (even if not merged).
 
+                                         *ddu-source-git_branch-action-create*
+create
+	Create new branch.
```

Comment:
```suggestion
	Create new branch based on the HEAD currently points.
```

In the ddu, "creates" action sounds like that I can expect it may create from the "selected" branch.
So I want it clear to make a new branch from "Current HEAD".

(I may create new action "copy" that creates new branch from "selected" one)